src/DevYeah.LMS/DevYeah.LMS.Web/app/src/features/identity/SignUpPage.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react';
+import SignUpForm from './SignUpForm';
+import './identity.css';
+
+export default class SignUpPage extends Component {
+  constructor(props){
+    super(props);
+
+    this.submitHandler = this.submitHandler.bind(this);
+  }
+
+  submitHandler(values, actions){
+    setTimeout(() => {
+      console.log(JSON.stringify(values));
+      actions.setSubmitting(false);
+      actions.resetForm();
+    }, 1000);
```

Comment:
Why wait for 1 second?

Reply:
That is a temporary implementation that help me display the return values from register form. And it will be deleted further.